monitoring/roles/loki/files/values.yaml

```diff
@@ -0,0 +1,56 @@
+# Taken from https://raw.githubusercontent.com/digitalocean/Kubernetes-Starter-Kit-Developers/main/05-setup-loki-stack/assets/manifests/loki-stack-values-v2.5.1.yaml
+loki:
+  enabled: true
```

Comment:
Maybe we can add some values for the node affinity. In the dev cluster right now, I see that loki was deployed on a default node pool instead of the cloud services node pool.

Reply:
Yah I wanted to add it but didn't see the options as clearly defined, I'll look again

Reply:
Ie there was no node affinity field

Reply:
I think we can. Take a look at https://github.com/grafana/helm-charts/blob/loki-stack-2.5.1/charts/grafana/values.yaml#L239-L242 and I think we are using version 2.5.1 so we can override the affinity.

Reply:
Ah but I think I'm installing this chart https://github.com/grafana/helm-charts/blob/loki-stack-2.5.1/charts/loki-stack/values.yaml

Reply:
https://github.com/grafana/helm-charts/blob/main/charts/loki/values.yaml and this is the subchart I think I need, so you're still right hehe

Reply:
ok addressed; I didn't do the daemonset because then we'd lose logs, just did loki

Reply:
Yeah, I think the daemonset should be deployed on all nodes since it's a daemonset.